### PR TITLE
Translations improved

### PR DESCRIPTION
**Pull request by om64**

### CODE REVIEW

File: plugin/badge/Resources/translations/tools.fr.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-badges: "Gestion des badges de l'espace de travail"
-my_badges: "Badges de l'espace de travail"
+badges: 'Gestion des badges de l'espace de travail'
```
Comment (Elorfin) [inline plugin/badge/Resources/translations/tools.fr.yml]: Missing quote : `l'espace`

Comment (om64) [inline plugin/badge/Resources/translations/tools.fr.yml]: Corrected by now.

**Comment by LaurentGruber** [comment]
LGTM

**Comment by ngodfraind** [comment]
LGTM

**Comment by purplefish32** [comment]
@claroline/maintainers What needs to be done for this to be mergable ?

**Comment by stefk** [comment]
I think @ngodfraind is going to resolve the conflicts.

**Pull request closed without merging**